azkaban-exec-server/src/test/java/azkaban/execapp/event/RemoteFlowWatcherTest.java

```diff
@@ -128,26 +121,22 @@ public void testLevel1RemoteFlowWatcher() throws Exception {
     runner2Thread.start();
     runner2Thread.join();
 
-    FileUtils.deleteDirectory(workingDir1);
-    FileUtils.deleteDirectory(workingDir2);
-
     testPipelineLevel1(runner1.getExecutableFlow(), runner2.getExecutableFlow());
   }
 
-  @Ignore
   @Test
   public void testLevel2DiffRemoteFlowWatcher() throws Exception {
```

Comment:
What's the difference between this test and the previous one testLevel1RemoteFlowWatcher? The only difference I can see is that the test flow file has two additional jobs. 
Could you add a comment to describe the purpose of this test or remove it if it is not necessary?

Reply:
I haven't analyzed it to see what the impact is, but there's a difference between using one of:
- [exec1-mod.flow](https://github.com/azkaban/azkaban/blob/master/test/execution-test-data/exectest1/exec1-mod.flow)
- [exec1.flow](https://github.com/azkaban/azkaban/blob/master/test/execution-test-data/exectest1/exec1.flow)

Here's the diff:

```
diff test/execution-test-data/exectest1/exec1-mod.flow test/execution-test-data/exectest1/exec1.flow
33,35d32
<   },{
<     "source" : "job7",
<     "target" : "job9"
40,43d36
<   },
<   {
<     "source" : "job9",
<     "target" : "job10"
129,138d121
<     "id" : "job9",
<     "jobType" : "java",
<     "layout" : {
<       "level" : 0
<     },
<     "jobSource" : "job9.job",
<     "expectedRuntime" : 1
<   },
<   {
<     "propSource" : "prop2.properties",
```

I would assume it has some purpose.. I don't see why it should be remove without knowing if it really is redundant. But to me that's outside the scope of this PR. I'm just making the existing tests run.